deps/src/polymake.cpp

```diff
@@ -42,22 +44,15 @@ JLCXX_MODULE define_module_polymake(jlcxx::Module& polymake)
 
     polymake_module_add_array(polymake);
 
+    polymake_module_add_direct_calls(polymake);
+
     polymake.method("initialize_polymake", &initialize_polymake);
     polymake.method("application", [](const std::string x) {
         data.main_polymake_session->set_application(x);
     });
 
     polymake.method("shell_execute", [](const std::string x) {
-        // FIXME: tuples with strings are broken in cxxwrap
-        // return res;
-        // instead we return an array of a bool and three strings now
-        auto         res = data.main_polymake_session->shell_execute(x);
-        jl_value_t** output = new jl_value_t*[4];
-        output[0] = jl_box_bool(std::get<0>(res));
-        output[1] = jl_cstr_to_string(std::get<1>(res).c_str());
-        output[2] = jl_cstr_to_string(std::get<2>(res).c_str());
-        output[3] = jl_cstr_to_string(std::get<3>(res).c_str());
-        return jlcxx::make_julia_array(output, 4);
+        return data.main_polymake_session->shell_execute(x);
```

Comment:
This is now no more broken in cxxwrap?

Reply:
I believe support for tuples (with pointers) arrived here:
https://github.com/JuliaInterop/CxxWrap.jl/issues/133